### PR TITLE
[Guidos Gorgeous Lasagna] Rewrites

### PR DESCRIPTION
**Pull request by BethanyG**
Another round of rewrites.

1.  More shortening of both exercise and concept `introduction.md`
2. Additional edits to concept `about.md`
3. Additional edits to stub file
4. Additional edits to the `instruction.md` file

### CODE REVIEW

**Comment by BethanyG** [comment]
@iHiD @kytrinyx  - apologies for pining you both here.  There is no huge rush on these, but I definitely need some perspective, because I am beginning to doubt my writing and programming skills. I think this is the last rewrite I'll be doing , at least for a while.  Many many thanks in advance for any 👀  you can lend.

**Comment by kytrinyx** [comment]
@BethanyG Sorry it's taken so long to get to this. I _think_ this all looks like good changes. I definitely agree that shortening things up a bit in the start there is a good choice.

I would say... ship-it. If people are confused, they'll open a thread in the forum and that will be great information that we can act on later.

**Comment by BethanyG** [comment]
@kytrinyx   💜   Thanks - [there is a thread on the forum](https://forum.exercism.org/t/i-am-struggling-with-the-lasagna-exercise/2091), but I don't know that I agree with everything suggested.  This was my interpretation, hence the request for review from you and Jeremey.  But if this looks ok, I'll merge it once I get an approval.  Beyond that, maybe the influx of students for Analytical April will prompt additional complaints and suggestions....

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 1]
```diff
@@ -1,110 +1,100 @@
 # Introduction
 
-Python is a [dynamic and strongly][dynamic typing in python] typed [object-oriented][object oriented programming] programming language.
-It employs both [duck typing][duck typing] and [gradual typing][gradual typing], via [type hints][type hints].
-Imperative, declarative (e.g., functional), and object-oriented programming _styles_ are all supported, but internally [everything in Python is an object][everythings an object].
+Python is a [dynamic and strongly][dynamic typing in python] typed programming language.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:3]:
```suggestion
Python is a [dynamic and strongly typed][dynamic typing in python] programming language.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 2]
```diff
@@ -1,110 +1,100 @@
...
+It employs both [duck typing][duck typing] and [gradual typing][gradual typing] via [type hints][type hints].
 
-This exercise introduces 4 major Python language features: Name Assignment (_variables and constants_), Functions (_and the return keyword_), Comments, and Docstrings.
+While Python supports many different programming _styles_, internally **everything in Python is an [object][everythings an object]**.
+This includes numbers, strings, lists, and even functions.
+
+We'll dig more into what all of that means as we continue through the track.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:9]: Nice.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 3]
```diff
@@ -1,110 +1,100 @@
...
+
+This first exercise introduces 4 major Python language features: Name Assignment (_variables and constants_), Functions (_and the return keyword_), Comments, and Docstrings.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:11]: Also nice.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 4]
```diff
@@ -1,110 +1,100 @@
...
-## Name Assignment and Re-assignment
 
-In Python, there are no keywords to define variables or constants.
-Both are [_names_][facts-and-myths-about-python-names] that help programmers reference values (_objects_) in a program and are written differently only by convention.
-On Exercism, [variables][variables] are always written in [`snake_case`][snake case], and _constants_ in `SCREAMING_SNAKE_CASE`.
+## Name Assignment (Variables & Constants)
 
-Names are assigned to values using `=`, or the [_assignment operator_][assignment statements] (`<name> = <value>`).
-A name (_variable or constant_) can be re-assigned over its lifetime to different values/object types.
+Programmer defined [_names_][facts-and-myths-about-python-names] (also called _variables__) can be bound to any type of object using the assignment `=` operator: `<name> = <value>`.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:27]:
```suggestion
Programmer-defined [_names_][facts-and-myths-about-python-names] (also called _variables__) can be bound to any type of object using the assignment `=` operator: `<name> = <value>`.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 5]
```diff
@@ -1,110 +1,100 @@
...
+A name can be reassigned (or re-bound) to different values (different object types) over its lifetime.
 
-For example, `my_first_variable` can be re-assigned many times using `=`, and can refer to different object types with each re-assignment:
 
 ```python
-# Assigning my_first_variable to a numeric value.
->>> my_first_variable = 1
+>>> my_first_variable = 1  # Name bound to an integer object of value one.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:32]:
```suggestion
>>> my_first_variable = 1  # my_first_variable bound to an integer object of value one.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 6]
```diff
@@ -1,110 +1,100 @@
...
+>>> my_first_variable = 2  # Name re-assigned to integer value 2.
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:33]:
```suggestion
>>> my_first_variable = 2  # my_first_variable re-assigned to integer value 2.
```

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md [hunk 7]
```diff
@@ -1,110 +1,100 @@
...
-
+The `def` keyword begins a [function definition][function definition].
+Each function can have zero or more formal [parameters][parameters] in `()` parenthesis, followed by a `:` colon.
 Statements for the _body_ of the function begin on the line following `def` and must be _indented in a block_.
-There is no strict indentation amount (_either space **OR** [tab] characters are acceptable_), but [indentation][indentation] must be _consistent_ for all indented statements.
 
 
 ```python
-# Function definition on first line.
+# The body of this function is indented by 2 spaces, & prints the sum of the numbers
```
Comment (iHiD) [inline exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:65]:
```suggestion
# The body of a function is indented by 2 spaces, & prints the sum of the numbers
```

**Review by iHiD — APPROVED** [review]
I really like these changes. Nice work :) I've made a few comments just for where things weren't immediately clear to me, in the hope of making them clearer to others. But you judge whether they're good suggestions or not :)

**Comment by iHiD** [comment]
I didn't see this until now. Sorry.

This is the current state of my GH notifications:

Screenshot 2023-03-03 at 18 47 08

@BethanyG Definitely always feel free to ping me on Slack or on the forum 1-1 if you'd like me to review something as there's a almost certain chance I'll miss things via my notifications! Always happy to be of help to you if I can 💙

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md (hunk 1 above)
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md:3]: _thank you_

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md (hunk 4 above)
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md]: Resolved this one on the branch.

File: exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md (hunk 6 above)
Comment (BethanyG) [reply exercises/concept/guidos-gorgeous-lasagna/.docs/introduction.md]: Thank you.  This reads much better now!

**Comment by BethanyG** [comment]
Whelp, let's hope this helps.  The stats have now sunk to 42.99% from 43.1%!!  😱 .  Thanks again to the both of you for the review! 💜

**Pull request merged**